skt/reporter.py

```diff
@@ -442,11 +430,9 @@ def report(self):
         print self.getreport()
 
         for (name, att) in self.attach:
```

Comment:
Should we also remove parentheses here?

Reply:
If I checked the code correctly, in the both cases you mention we use lists of tuples. While in these cases the parens are not necessary for correct value unpacking, I left them there for the clarity that the iteration happens on tuples. So you can automatically see what the variable contains to work with, instead of needing to check the initialization, or having to add a comment there (I'm not a fan of comments where the functionality can be made clear form the code).

I don't feel strong about either way (and `pylint` should be fine with both too, AFAIK), it's just a readability thing since the structures used are a bit complicated.

Reply:
Ah, I see, that's fine by me :)